.vib/mediawiki/goss/goss.yaml

```diff
@@ -0,0 +1,23 @@
+file:
```

Comment:
There is a need for one more test case here, in order to meet the 5 integration tests acceptance criteria.
This application is not really Goss friendly. I hope there are some other interesting files to verify.

Reply:
Added a check for the php binary used, as we have had cases in the past where the PHP in $PATH was the one installed manually by users.